src/js/utils/notification-utils.jsx

```diff
@@ -0,0 +1,17 @@
+import queryString from 'query-string';
+
+import notification from 'components/Layout/notifications/notification';
+import NotificationType from 'consts/notificationTypes';
+import translate from 'utils/Translate';
+
+export default function triggerDeletionNotification(queryParams) {
+  const parsedQueryParams = queryString.parse(queryParams);
+  if (parsedQueryParams.deleted) {
+    notification(NotificationType.SUCCESS)({
+      message: translate({
+        id: 'react.stockMovement.deleted.success.message.label',
+        defaultMessage: 'Stock Movement has been deleted successfully',
+      }),
+    });
+  }
+}
```

Comment:
As I have mentioned in my first comment about generalizing this utils function it would look  something like that
```js
export default function triggerNotification(queryParams) {
  const parsedQueryParams = queryString.parse(queryParams);
  const { notificationMessage, notificationType } = parsedQueryParams
  if (notificationMessage) {
    notification(notificationType || NotificationType.SUCCESS)({
      message: notificationMessage,
    });
  }
}
```